package.json

```diff
@@ -44,4 +45,4 @@
   "engines": {
     "node": ">=18.0"
   }
-}
+}
```

Comment:
Not nice to have “no newline at end of file”. Note that it is a good style to always put the newline as a last character, in text files, if it is allowed by the file format.

Reply:
This won't happen again